chain-indexer/src/infra/storage/postgres.rs

```diff
@@ -271,7 +280,27 @@ async fn save_transactions(
         .await?;
 
     for (transaction, transaction_id) in transactions.iter().zip(transaction_ids.iter()) {
-        save_contract_actions(&transaction.contract_actions, *transaction_id, tx).await?;
+        let contract_action_ids =
+            save_contract_actions(&transaction.contract_actions, *transaction_id, tx).await?;
+
+        let contract_balances: Vec<(i64, ExtractedBalance)> = transaction
```

Comment:
Remove this type annotation. Instead turbo fish the collect; I guess `collect::<Vec<_>>()` is enough.